SoftLayer/CLI/hardware/create.py

```diff
@@ -129,11 +133,15 @@ def cli(env, **args):
 
         result = mgr.place_order(**order)
 
-        table = formatting.KeyValueTable(['name', 'value'])
-        table.align['name'] = 'r'
-        table.align['value'] = 'l'
-        table.add_row(['id', result['orderId']])
-        table.add_row(['created', result['orderDate']])
-        output = table
+        if args['output_json']:
+            env.fout(json.dumps({'orderId': result['orderId']}))
```

Comment:
It's odd that specifying json causes different content to be output, not just a different format. It looks like below you get `id: orderId, created: orderDate`, but here you get `orderId: orderId`. The key names are different and there's a missing key.